utils/oscap_docker_python/oscap_docker_util.py

```diff
@@ -120,16 +120,19 @@ def oscap_chroot(self, target, image, chroot_path, *oscap_args):
                    "PRIMARY_HOST_NAME"] = "{0}-{1}".format(target, image)
         cmd = ['oscap'] + [x for x in oscap_args]
         try:
-            run = subprocess.check_output(cmd)
-        except Exception as error:
-            print("\nCommand: {0} failed!\n".format(" ".join(cmd)))
-            print("Error was:\n")
-            print(error)
-
-            # Clean up
-            self._cleanup_by_path(chroot_path)
-
-            sys.exit(1)
+            run = subprocess.check_output(cmd, stderr=subprocess.STDOUT)
```

Comment:
This looks like a hack, should we be using `subprocess.call` and checking whether exit_code is in [0, 2]? It gives us one way to get stderr and one way to get stdout. I think it would be cleaner.